Suppress noisy clippy warning with Rust 1.93
Can go away once ctest 0.5.0 is released.

Index: systest/src/main.rs
--- systest/src/main.rs.orig
+++ systest/src/main.rs
@@ -1,4 +1,4 @@
-#![allow(bad_style, deprecated, clippy::all)]
+#![allow(bad_style, deprecated, function_casts_as_integer, clippy::all)]
 
 use libc::*;
 use openssl_sys::*;
